dist/README.md

```diff
@@ -0,0 +1,27 @@
+# Installing
+
+Pick the binary that matches your operating system and architecture. For
+example, on my Macbook Pro I would use `ftree-darwin-amd64`. Copy that somewhere
+in your path and rename to `ftree`. Presto bango.
+
+## Verifying
+
+The best way to verify the release is to examine the GPG signature. You can find
+my public key at http://calebthompson.io/pubkey.asc,
+<https://pgp.mit.edu/pks/lookup?op=get&search=0x1621ADC2A0ACE70A>, and via GnuPG
+with `gpg --recv-keys A0ACE70A`.
```

Comment:
Further, a SHA sum would be easier to fake as the signature is shorter. Here's the signature on `ftree-darwin-amd64`:

```
-----BEGIN PGP SIGNATURE-----

iQEcBAABCgAGBQJWFrX8AAoJEBYhrcKgrOcKkvgH/3hMp0JywCLSFLfPMIF0Cca8
WvpMcKsR3WY4s5iUm1djdb8Bp2eLxhFzWe78JM+KxAafuc500CjbLXdoF/LISDM7
Be2lFjdOdDwpMD5UcJDoj4wUE2wnHKAdPR36zlNhLNV+qSbnbb16khY6ZFTeXPZE
f2Cml3GxcCHag6bKKjy1GTZacDiBKTeOgf1lkFmqyccWAPdCO+5qeLs2dGbubt36
v3KU8JMOjIXFVbEJi+P09vuv+NWT9QJvnp5ZepQlGqLoWvMynUpMO7Y84u4+4Ca9
b05/W+WkFGcwWvZGiyBcXzAGUMyiKuvtsjGW/XOz/MB5igfhIJqQX2nN5lvYoJQ=
=IcUU
-----END PGP SIGNATURE-----
```

And the SHA sum:

```
a7342c0f695e229092aa304e2ffd7908035e9ff6e237acdf4650e2783c862212  ftree-darwin-amd64
```